RegularApi.Tests/Dao/DeploymentTemplateDaoIT.cs

```diff
@@ -0,0 +1,55 @@
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using RegularApi.Dao;
+using RegularApi.Tests.Fixtures;
+
+namespace RegularApi.Tests.Dao
+{
+    public class DeploymentTemplateDaoIT : BaseDaoIT
+    {
+        private IDeploymentTemplateDao _templateDao;
```

Comment:
I would change the name of the property to `_deploymentTemplateDao` but it's not a big deal